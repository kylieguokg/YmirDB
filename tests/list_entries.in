LIST ENTRIES
SET a 28 19 10 402 1 2
SET b 0 28 10 a
LIST ENTRIES
SET c c
SET c 0
LIST ENTRIES
set c m
LIST ENTRIES
BYE
