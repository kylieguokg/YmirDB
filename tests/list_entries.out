> no entries

> ok

> ok

> b [0 28 10 a]
a [28 19 10 402 1 2]

> not permitted

> ok

> c [0]
b [0 28 10 a]
a [28 19 10 402 1 2]

> no such key

> c [0]
b [0 28 10 a]
a [28 19 10 402 1 2]

> bye
